vendor/assets/stylesheets/dvl/core/forms.scss

```diff
@@ -83,6 +83,20 @@ textarea {
   }
 }
 
+// Focus states
+
+input[type="text"],
+input[type="email"],
+input[type="search"],
+input[type="password"],
+input[type="url"],
+textarea {
+  &:focus {
+    border-color: $darkBlue;
```

Comment:
Name of variable `darkBlue` should be written in all lowercase letters with hyphens instead of underscores